packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
```

Comment:
We could probably remove this page level "use client" and move it to `MemberList.tsx` just to get benefits of server component and also improves [Initial Page load + FCP](https://arc.net/l/quote/fpaxbdti)